DigitalLearningSolutions.Data/Services/TutorialContentService.cs

```diff
@@ -49,17 +49,39 @@ int tutorialId
                          Tutorials.Objectives,
                          Tutorials.VideoPath,
                          Tutorials.TutorialPath,
-                         Tutorials.SupportingMatsPath AS SupportingMaterialPath
+                         Tutorials.SupportingMatsPath AS SupportingMaterialPath,
+                         Sections.PLAssessPath AS PostLearningAssessmentPath,
+                         MIN(SubsequentTutorials.TutorialID) AS NextTutorialID,

```

Comment:
Does this return the next tutorial according to the OrderByNumber? 🙂

Reply:
Yes it does, I added that as a comment in bc4cce2 🙂